ios/RIBs/Classes/Presenter.swift

```diff
@@ -19,7 +19,7 @@ import Foundation
 /// The base protocol for all `Presenter`s.
 public protocol Presentable: class {}
 
-/// The base class of all `Pesenters`. A `Pesenters` translates business models into values the corresponding
+/// The base class of all `Presenters`. A `Presenters` translates business models into values the corresponding
```

Comment:
This should actually be:

The base class of all `Presenter`s. A `Presenter` translates business models into values the corresponding